lightning/pkg/importer/get_pre_info.go

```diff
@@ -428,15 +428,15 @@ func newTableInfo(createTblSQL string, tableID int64) (*model.TableInfo, error)
 		log.L().Error(errMsg, zap.Error(err), zap.String("sql", createTblSQL))
 		return nil, errors.Trace(err)
 	}
-	sctx := mock.NewContext()
 	createTableStmt, ok := astNode.(*ast.CreateTableStmt)
 	if !ok {
 		return nil, errors.New("cannot transfer the parsed SQL as an CREATE TABLE statement")
 	}
-	info, err := ddl.MockTableInfo(sctx, createTableStmt, tableID)
+	info, err := ddl.BuildTableInfoFromAST(metabuild.NewNonStrictContext(), createTableStmt)
```

Comment:
Seems `newTableInfo` always use `0` as table it and it is not actually used when importing data:

https://github.com/pingcap/tidb/blob/b427e33eb9e0479e6a4000e0b45f04144a79dc2a/lightning/pkg/importer/get_pre_info.go#L411

PTAL @lance6716